lib/Worker.js

```diff
@@ -74,8 +75,10 @@ module.exports = (function(){
                             return liveEntry;
                         })
                         .catch(function(err) {
-                            err.entry = liveEntry;
-                            logger.debug("Live entry [%s] Failed to start", e.entryId);
+                            logger.error("Live entry [%s] Failed to start", e.entryId);
+                            if (_.isObject(err)){
```

Comment:
Please move [%s] to beginning of string to be consistent with other logger printings
,